README.md

```diff
@@ -2351,6 +2352,44 @@ Checks if string ends with the given target string.
 
 **[⬆ back to top](#quick-links)**
 
+### _.isString
+:heavy_exclamation_mark:`Not in Underscore.js`
+Checks if string ends with the given target string.
+
+  ```js
+  // Lodash
+  _.isString('abc');
+  // => true
+ 
+  _.isString(123);
+  // => false
+  
+  _.endsWith('abc', 'b', 2);
+  // => true
+
+  // Native
+  function isString(str){
+    if (str && typeof str.valueOf() === "string") {
+      return true
+      }
+      return false
+  }
+ 
+  isString('abc');
+  // => true
+ 
+  isString(123);
+  // => false
+  ```
+
+#### Browser Support for `String.prototype.endsWith()`
+
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: | :-: |
+  41.0 ✔  |  ✔ | 17.0 ✔ |  ✖  |  28.0 ✔ |  9.0 ✔ |
```

Comment:
Why did you add `X` in IE?

Reply:
let me get that fixed

Reply:
Same above, you forgot update browser support

Reply:
@NamPNQ @ODudek  updated

Reply:
i  will be doing another pr as soon as it gets resolved